lang/cs/Org.Apache.REEF.Common/Org.Apache.REEF.Common.csproj

```diff
@@ -56,18 +56,18 @@ under the License.
     <Compile Include="Api\IFailure.cs" />
     <Compile Include="Attributes\UnstableAttribute.cs" />
     <Compile Include="Avro\AvroDriverInfo.cs">
-        <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
+      <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
```

Comment:
VS changed it for me, but this seems to be the right indentation format instead of the original one.